examples/network-manager-app/linux/args.gni

```diff
@@ -22,3 +22,4 @@ chip_project_config_include_dirs = [
 ]
 
 chip_config_network_layer_ble = false
+chip_enable_access_restrictions = true
```

Comment:
Putting a comment on a random file so it doesn't get lost. This is a really big change (25 files) to a cluster that is key to our security model. We need to give reviewers a chance to look this over correctly and we're right in the middle of a test event. Please wait on submitting this PR until folks have a reasonable chance to review (ie after the end of TE2, with a couple of days to allow sufficient reviews)